setup.sh

```diff
@@ -41,10 +41,11 @@ declare -r DOCUMENTATION='documentation'
 
 declare -r CONFIGS_PATH='configs'
 
+declare -r DOCS_VENV='docs_venv'
```

Comment:
What the `V` in `VENV`  means? Can we have a more readable name?

Reply:
`V` stands for "virtual". I will change it.